build.xml

```diff
@@ -26,16 +34,33 @@
         </fileset>
     </path>
 
+    <available file="${ivy.lib}/${ivy.jar}" property="ivy.exists"/>
+    
+    <target name="bootstrap" description="Install ivy" unless="ivy.exists">
+        <mkdir dir="${ivy.lib}"/>
+        <get dest="${ivy.lib}/${ivy.jar}" src="http://search.maven.org/remotecontent?filepath=org/apache/ivy/ivy/${ivy.version}/${ivy.jar}"/>
+    </target>
+    
+    <target name="resolve" description="Use ivy to resolve classpaths" depends="bootstrap">
```

Comment:
Ant returns:

> BUILD FAILED build.xml:46: The prefix "ivy" for element "ivy:resolve" is not bound.

I'd suggest adding `xmlns:ivy="antlib:org.apache.ivy.ant"` to the root `<project>` element or to the `<target>` that invokes the ivy tasks.

Reply:
Okay here goes again...